dev/breeze/src/airflow_breeze/utils/docker_command_utils.py

```diff
@@ -299,6 +299,38 @@ def check_docker_compose_version(verbose: bool):
         )
 
 
+def check_docker_context(verbose: bool):
+    """
+    Checks whether Docker is using the expected context
+    :param verbose: print commands when running
+    """
+    expected_docker_context = "default"
+    response = run_command(
+        ["docker", "info", "--format", "{{json .ClientInfo.Context}}"],
+        verbose=verbose,
+        no_output_dump_on_exception=False,
+        text=True,
+        capture_output=True,
+    )
+    if response.returncode != 0:
+        get_console().print(
+            '[error]Could not check for Docker context.[/]\n'
```

Comment:
I'd return into warning and immediately return from the function rather than print error. Generally as a rule "error" is reserved for some things that the user should fix to continue. 

We have this ADR written describing it: https://airflowsummit.org/sessions/2021/dynamic-security-roles-in-airflow-for-multi-tenancy/

In this case, if we cannot check docker, I think this might mean for example that for whatever reason docker "client' we have does not provide this information (but in this case I'd say things should work in general - so likely we can continue). 

The following warning is also a bit misleading - what does it mean "installed and running properly" ? The message should provide some helpful hints to the user - for example "properly (`docker info` should show Context: default")

Reply:
You're totally right :) 
We should return here because it's not making sense checking it further, and also I changed the message to be more actionable.